app/src/main/java/com/example/fittracker/MainActivity.java

```diff
@@ -2,12 +2,44 @@
 
 import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
 
 public class MainActivity extends AppCompatActivity {
-
+    private LogInContract.Presenter presenter;
+    private EditText mail;
+    private EditText password;
+    private UserBase lista;
+    private final String MAIL_PRUEBA = "danko94.cg@gmail.com";
+    private final String PASS_PRUEBA = "1234";
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
+        mail=findViewById(R.id.mail_usuario);
+        password=findViewById(R.id.password);
+        lista = new UserBase();
+        lista.addUsuario(new User(MAIL_PRUEBA,PASS_PRUEBA));
```

Comment:
all this init data must be in the presenter

Reply:
Done